### PR TITLE
Update release note

### PR DESCRIPTION
**Pull request by MuyangAmigo**

### CODE REVIEW

File: packages/vscode-extension/PRERELEASE.md [hunk 1]
```diff
@@ -1,7 +1,38 @@
...
+
+- Updated `React with Fluent UI` app template to use OBO flow for SSO implenmentation to improve SSO experience in mobile platforms.
+- Updated project scaffold success notification for Outlook Add-in.
+
+TeamsFx SDK ([`@microsoft/teamsfx`](https://www.npmjs.com/package/@microsoft/teamsfx)) `v2.2.2`:
+
+- Rename existing internal class `ConversationReferenceStore` to `DefaultConversationReferenceStore`.
+- Deprecate existing interface `NotificationStorage`.
+- Add new interface `ConversationReferenceStore` with pagination API support.
+- Add new options for users to set customized conversation reference store, and deprecate existing option to set customize notification target storage.
```
Comment (therealjohn) [inline packages/vscode-extension/PRERELEASE.md:24]: Added a new option to set a custom `ConversationReferenceStore` and deprecated the existing option for `NotificationStorage`.

Where do you set the custom `ConversationReferenceStore`?

File: packages/vscode-extension/PRERELEASE.md [hunk 2]
```diff
@@ -1,7 +1,38 @@
...
+New features:
+
+- Import an existing SharePoint Framework solution and continue development with Teams Toolkit
+    ![SPFx Existing App](https://github.com/OfficeDev/TeamsFx/assets/11220663/3944f5c8-6c8c-4b4d-8df8-dc4f45b5967f)
+- A new link unfurling app template to help you get started with displaying rich content from links in Teams messages and Outlook emails.
+    ![Link Unfurling](https://github.com/OfficeDev/TeamsFx/assets/11220663/6e8b982a-0531-4ec1-8420-f6f17955ff40)
+
+Enhancement:
+
+- Updated `React with Fluent UI` app template to use OBO flow for SSO implenmentation to improve SSO experience in mobile platforms.
```
[the remark this answers is not in the document]
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md:16]: Updated

File: packages/vscode-extension/PRERELEASE.md [hunk 3]
```diff
@@ -1,7 +1,38 @@
...
+- Updated project scaffold success notification for Outlook Add-in.
+
+TeamsFx SDK ([`@microsoft/teamsfx`](https://www.npmjs.com/package/@microsoft/teamsfx)) `v2.2.2`:
+
+- Rename existing internal class `ConversationReferenceStore` to `DefaultConversationReferenceStore`.
```
[the remark this answers is not in the document]
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md:21]: Updated

File: packages/vscode-extension/PRERELEASE.md [hunk 4]
```diff
@@ -1,7 +1,38 @@
...
+- Deprecate existing interface `NotificationStorage`.
```
[the remark this answers is not in the document]
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md:22]: Updated.

File: packages/vscode-extension/PRERELEASE.md [hunk 5]
```diff
@@ -1,7 +1,38 @@
...
+- Add new interface `ConversationReferenceStore` with pagination API support.
```
[the remark this answers is not in the document]
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md:23]: Updated

File: packages/vscode-extension/PRERELEASE.md [hunk 6]
```diff
@@ -1,7 +1,38 @@
...
+- Add new options for users to set customized conversation reference store, and deprecate existing option to set customize notification target storage.
+
+Teams Toolkit CLI ([`@microsoft/teamsfx-cli`](https://www.npmjs.com/package/@microsoft/teamsfx-cli)) `v2.0.2@beta`:
+
+- (Feature parity with Visual Sudio Code) Import an existing SPFx solution and continue development with Teams Toolkit.
+- (Feature parity with Visual Sudio Code) Create a new link unfurling app with Teams Toolkit.
+
+Bug fixes:
+
+- Fixed an issue where code lens to open Adaptive Card will only be available for cards under `adaptiveCards` folder. ([#9232](https://github.com/OfficeDev/TeamsFx/pull/9232))
```
[the remark this answers is not in the document]
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md:33]: Updated.

**Review by sffamily — APPROVED** [review]

**Review by yukun-dong — DISMISSED** [review]

**Pull request merged**